### DISCUSSION TITLE
Allow Saving Queue Line TVs to Ride Track Design

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17824] [Feature requests] [answered]
Marcel Vos is speedrunning openrct2, but he cannot save Queue Line TVs to TD6, it would be nice if we could so if anyone speedruns in the future they don't have to manually place queue line tvs
example : 
![Github Park 2022-08-14 16-47-10](https://user-images.githubusercontent.com/94884086/184554425-b111781d-ed95-4634-ae86-f0a597410a5d.png)
![Github Park 2022-08-14 16-47-04](https://user-images.githubusercontent.com/94884086/184554427-f69b13dd-ea3d-40bb-b87a-e1ba36a73b8f.png)

**Comment by Broxzier** [comment, marked as answer]
Not being able to save path bits is a known technical restriction of the TD6 format.
